news/2024.html

```diff
@@ -30,6 +30,57 @@
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="summit-2024-keynotes">
+  <a href="#summit-2024-keynotes"><span class="tag-span"># </span></a> 12 June 2024: Keynotes for seL4 summit 2023 announced
+</div>
+<div>
+  <div class="row" style="margin-left:0px; margin-right:0px">
+  <p>
+    We are pleased to announce that the two keynotes for the <a href="../Foundation/Summit/2024">seL4 summit 2024</a> will be <strong>Darren Cofer from Collins Aerospace</strong> and <strong>Ning Qu from
```

Comment:
I suggest <strong>Darren Cofer</strong> from <strong>Collins Aerospace</strong>
and add a link to Darren's page/linkedIn from his name, and a link to Collins Aerospace from the company name.
Same for Ning and NIO.